app.js

```diff
@@ -62,29 +29,69 @@ var getCookiesPerHour = function(objectName){
   }
 };
 
+CookieStore.prototype.makeTheTable = function(tableToDrawInto){
+  var hourlySalesRowEl = document.createElement('tr');
+  console.log(hourlySalesRowEl);
+  var storeNameEl = document.createElement('th');
+  console.log(storeNameEl);
+  storeNameEl.textContent = this.name;
+  hourlySalesRowEl.appendChild(storeNameEl);
+
+  for(var hours = 0; hours < this.cookiesPerHour.length; hours++){
+    var hourlyCookieSales = document.createElement('td');
+    console.log(hourlyCookieSales);
+    hourlyCookieSales.textContent = this.cookiesPerHour[hours];
+    console.log(hourlyCookieSales);
+    hourlySalesRowEl.appendChild(hourlyCookieSales);
+  }
+  tableToDrawInto.appendChild(hourlySalesRowEl);
+};
+
 //Call a function for each city to populate the arrays in each object.
 getCookiesPerHour(firstAndPike);
 getCookiesPerHour(seatacAirport);
 getCookiesPerHour(seattleCenter);
 getCookiesPerHour(capitolHill);
 getCookiesPerHour(alki);
 
-//Create function to create an ul and insert the array into it.
-function putSalesProjections(location){
-  console.log(location.name + '-projections');
-  var ulElement = document.getElementById(location.name + '-projections');
+var tableEl = document.createElement('table');
+console.log(tableEl);
+var sectionEl = document.getElementById('cookieSalesTable');
+sectionEl.appendChild(tableEl);
 
-  for(var ii = 0; ii < 15; ii++){
-    var liElement = document.createElement('li');
-    liElement.setAttribute('class', 'avgCookiesPerHour');
-    liElement.innerHTML = storeHours[ii] + ': ' + location.cookiesPerHour[ii] + ' cookies';
-    ulElement.appendChild(liElement);
+var hoursRowEl = document.createElement('tr');
+var hoursHeaderEl = document.createElement('th');
+hoursRowEl.appendChild(hoursHeaderEl);
+for(var tableHours = 0; tableHours < storeHours.length; tableHours++){
+  var hoursDataEl = document.createElement('td');
+  hoursDataEl.textContent = storeHours[tableHours];
+  hoursRowEl.appendChild(hoursDataEl);
+}
+tableEl.appendChild(hoursRowEl);
+
+firstAndPike.makeTheTable(tableEl);
+seatacAirport.makeTheTable(tableEl);
+seattleCenter.makeTheTable(tableEl);
+capitolHill.makeTheTable(tableEl);
+alki.makeTheTable(tableEl);
+
+var stores = [firstAndPike, seatacAirport, seattleCenter, capitolHill, alki];
```

Comment:
Maybe group this line with where you instantiate those instances of CookieStore.